.changeset/young-pigs-live.md

```diff
@@ -0,0 +1,12 @@
+---
+'@clerk/backend': minor
+'@clerk/nextjs': minor
+'@clerk/shared': minor
+---
+
+Introduces dynamic keys from `clerkMiddleware`, allowing access by server-side helpers like `auth`. Keys such as `signUpUrl`, `signInUrl`, `publishableKey` and `secretKey` are securely encrypted using AES algorithm.
```

Comment:
```suggestion
Introduces dynamic keys from `clerkMiddleware`, allowing access by server-side helpers like `auth`. Options such as `signUpUrl`, `signInUrl`, `publishableKey` and `secretKey` are securely encrypted using AES algorithm and made available to the Next.js backend.
```

Reply:
I was getting confused between "Keys" and "Options", nice feedback!